psl-clj/.gitignore

```diff
@@ -0,0 +1,11 @@
+*jar
```

Comment:
Move all the common ones (like *.*~) up to the project root.
Then have only the Clojure specific ones here.
So ones like lib/target/classes will probably be unnecessary.

Reply:
Done